mevislab.github.io/content/tutorials/basicmechanisms.md

```diff
@@ -71,79 +68,78 @@ You are not restricted to only see your image in 2D. The Output Inspector offers
 
-4.  At the input connector of `View2D`, release the mouse button and establish the connection (blue line).
+4.  Release the mouse-button on the input connector of your 'View2D'-module to establish the connection. 
 
 ![Establish connection](/images/tutorials/basicmechanics/BM_06.png "Establish connection")
 
-Although the connection is established, no image rendering has started yet. To initialize rendering, open the `View2D` panel by double-clicking the module `View2D` in your network. Similar to the Output Inspector, you can scroll through the slices and change the contrast. By pressing the {{< keyboard "A" >}} you can switch the annotation mode.
+Although the connection is established, no image rendering has started yet. To initialize rendering, open the `View2D` panel by double-clicking on the module. Similar to the Output Inspector, you can scroll through the slices and set different levels of contrast. The amount of displayed annotations is altered by pressing {{< keyboard "A" >}} on the keyboard (annotation-mode).
 
 ![View2D Panel](/images/tutorials/basicmechanics/BM_07.png "View2D Panel")
 
-You can also disconnect the Modules by dragging the connection away from the input or output connector of the Modules.
+By dragging the connection away from either the input or the output connector, the connection is interrupted.
 
-Connections between compatible outputs and inputs are established automatically in case 2 Modules are moved close enough to each other.
+Connections between compatible outputs and inputs are established automatically if two Modules get close enough to each other.
 
 {{<alert class="info" caption="Extra Infos">}}
-Connecting, Disconnecting, Moving and Replacing Connections is explained {{< docuLinks "/Resources/Documentation/Publish/SDK/MeVisLabManual/ch03s04.html" "here" >}}
+Connecting, Disconnecting, Moving and Replacing Connections is furtherly explained {{< docuLinks "/Resources/Documentation/Publish/SDK/MeVisLabManual/ch03s04.html" "here" >}}
 {{</alert>}}
 
 [//]: <> (MVL-653)
 
 ### Image Processing {#TutorialImageProcessing}
 
-Next, we like to add a processing step to our network. We like to smooth the image using an average kernel. In order to do that, search for the module `Convolution` and add it to the workspace. Delete the established connection from the module `ImageLoad` to the module `View2D` by clicking on the connection and pressing {{< keyboard "DEL" >}}. Now, you can build new connections from the module `ImageLoad` to the module `Convolution` and from this module to `View2D`.
+An average kernel will be used to smooth the image out as our next step will be to actually process our image. Add the `Convolution`-Module to your workspace and disconnect the 'View2D'-Module from the 'ImageLoad'-Module by clicking on the connection and pressing {{< keyboard "DEL" >}}. Now, you can build new connections from the module `ImageLoad` to the module `Convolution` and the 'Convolution'-Module to `View2D`.
```

Comment:
`Convolution` module, `View2D` module, `ImageLoad` module, the same further down